packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -262,7 +263,7 @@ const VideoPlayerBase = class extends React.Component {
 		 * @default `moonstone/VideoPlayer.MediaControls`
 		 * @public
 		 */
-		mediaControlsComponent: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
+		mediaControlsComponent: EnactPropTypes.componentOverride,
```

Comment:
Should we update the `@type` here as well?  I think it should be `{Component|Element}`.